bin/centreon_health

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env perl
+
+use warnings;
+use centreon::script::centreon_health;
+
+centreon::script::centreon_health->new()->run();
+
+__END__
+
+=head1 NAME
+
+centreon_health - a tool/script to gather informations about Centreon ("noroot" script)
+
+=head1 SYNOPSIS
+
+centreon_health [options]
+
+=head1 OPTIONS
+
+=over 8
+
+=item B<--centstorage-db> <value>
+
+Specify the name of your centstorage db if not standard
+(default: centreon_storage).
+
+=item B<--centreon-branch> <value>
```

Comment:
I think it could be better if you get this information in the centreon database directly.